node/channel.js

```diff
@@ -159,7 +159,7 @@ function TChannel(options) {
     }
 
     // populated by makeSubChannel
-    self.topChannel = null;
+    self.topChannel = self.options.topChannel || null;
```

Comment:
:warning: `self.options` should just be `options` to make less work of eliminating `self.options` if possible.